crates/sozo/ops/src/auth.rs

```diff
@@ -5,7 +5,7 @@ use dojo_world::utils::TransactionWaiter;
 use starknet::accounts::Account;
 
 use super::get_contract_address;
-use crate::commands::auth::{AuthCommand, AuthKind, ResourceType};
+use sozo::commands::auth::{AuthCommand, AuthKind, ResourceType};
```

Comment:
@glihm This is the dependency on sozo_ops for commands in sozo. This is the circular dependencies.

Reply:
Thanks for pointing this out. Will have a look how we can decouple that.

Reply:
@Jonatan-Chaverri yes so here, the things have to be decoupled.
The command related code (typically the match) should be done into command. But the `ops` must only run the code with provided inputs.

So to achieve the final result we're looking for, the `execute` function of `ops/src/auth.rs` should not take a `AuthCommand` as input.

You may also add some functions, like `grant_writer` / `grant_owner`. It was made with the switch all inside `execute` as it wasn't decoupled before. But here we try to decouple that for better testing + easy re-use of the `ops`.

Sounds good to you? Please don't hesitate if you need more info (we can hoop in a call). :+1:

Reply:
@glihm I tried to follow what you are saying, please check if this is what you meant. Also not sure how to decouple ResourceType Enum from sozo -> commands -> auth.rs in the crates auth file. Can you please provide some inputs on that, should we move the enums to crates files?

Reply:
Also, same issue is happening on events.rs, execute.rs, models.rs and register.rs... all of them have dependencies on commands and needs to be decoupled.